app/controllers/forms_controller.rb

```diff
@@ -10,6 +10,11 @@ def show
     @task_list = FormTaskListService.call(form: @form).all_tasks
   end
 
+  def append_info_to_payload(payload)
+    super
+    payload[:form_id] = params[:form_id]
+  end
```

Comment:
I think we can structure this a bit better. 

Its odd that we have a "FormsController" and and "FormsBaseController". We had this issue with forms-runner and refactored it to have a single forms base controller that things would inherit from.

Reply:
yes - could I make the append_info_to_payload in application_controller add form_id if it's in params - then it doesn't need to be repeated? Do you think it's ok to take it from the params like this, rather than the From model from API?

I think the FormsController" and "FormsBaseController" is unfortunate naming in this case - they are not related. The FormsController handles showing the `tasklist`, `FormsBaseController` is the base class for the forms pattern, maybe we can rename the forms pattern controllers into something which doesn't clash with form?